### PR TITLE
feat(.github): support @kumahq for PR comments

### PR DESCRIPTION
**Pull request by michaelbeaumont**
It was kinda bugging me

### Checklist prior to review

- [ ] Link to docs PR or issue --
- [ ] Link to UI issue or PR --
- [ ] Is the [issue worked on linked][1]? --
- [ ] The PR does not hardcode values that might break projects that depend on kuma (e.g. "kumahq" as a image registry) --
- [ ] The PR will work for both Linux and Windows, system specific functions like `syscall.Mkfifo` have equivalent implementation on the other OS --
- [ ] Unit Tests --
- [ ] E2E Tests --
- [ ] Manual Universal Tests --
- [ ] Manual Kubernetes Tests --
- [ ] Do you need to update [`UPGRADE.md`](../blob/master/UPGRADE.md)? --
- [ ] Does it need to be backported according to the [backporting policy](../blob/master/CONTRIBUTING.md#backporting)? --
- [ ] Do you need to explicitly set a [`> Changelog:` entry here](../blob/master/CONTRIBUTING.md#submitting-a-patch) or add a `ci/` label to run fewer/more tests?

[1]: https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword

### CODE REVIEW

**Review by kleinfreund — APPROVED** [review]

**Comment by michaelbeaumont** [comment]
I just want to change the format to `@kumahq` instead of `/` :shrug:

File: .github/workflows/pr-comments.yaml [hunk 1]
```diff
@@ -8,7 +8,7 @@ env:
 
 jobs:
   pr_comments:
-    if: github.event.issue.pull_request != '' && (contains(github.event.comment.body, '/format') || contains(github.event.comment.body, '/golden_files'))
+    if: github.event.issue.pull_request != '' && (contains(github.event.comment.body, '/format') || contains(github.event.comment.body, '/golden_files') || contains(github.event.comment.body, '@kumahq'))
```
[the remark this answers is not in the document]
Comment (michaelbeaumont) [reply .github/workflows/pr-comments.yaml:11]: The name of the account is `kumahq`, it's not necessarily the org owner.

File: .github/workflows/pr-comments.yaml [hunk 2]
```diff
@@ -57,11 +57,11 @@ jobs:
           make dev/tools
           git checkout go.mod go.sum # This is because of some weirdness in `make dev/tools`
       - name: format
-        if: contains(github.event.comment.body, '/format')   # check the comment if it contains the keywords
+        if: contains(github.event.comment.body, '/format') || contains(github.event.comment.body, '@kumahq format')
         run: |
           make format
       - name: run golden_files
-        if: contains(github.event.comment.body, '/golden_files')  # check the comment if it contains the keywords
+        if: contains(github.event.comment.body, '/golden_files') || contains(github.event.comment.body, '@kumahq golden files')
```
Comment (lahabana) [inline .github/workflows/pr-comments.yaml:64]: please keep command in 1 work `golden_files`

**Comment by lahabana** [comment]
Ok makes sense. Let's not do it nicely and just stop supporting `/` commands it will keep this simpler :)

**Comment by lahabana** [comment]
Also please update `developer.md` which I think references these

File: .github/workflows/pr-comments.yaml (hunk 2 above)
Comment (michaelbeaumont) [reply .github/workflows/pr-comments.yaml:64]: Any particular reason why? dependabot just has them separate

Comment (lahabana) [reply .github/workflows/pr-comments.yaml:64]: I'm afraid of conflicts but might be fine

**Review by lahabana — APPROVED** [review]
LGTM once you get rid of the old stuff

File: .github/workflows/pr-comments.yaml (hunk 2 above)
Comment (michaelbeaumont) [reply .github/workflows/pr-comments.yaml]: So what isn't supported here _anyway_ is something like `@kumahq format golden_files`. It'd require more advanced parsing of the comment body. You have to do: 

```
@kumahq format
@kumahq golden_files
```

**Comment by michaelbeaumont** [comment]
Closing this for now, I'm not happy with the @ version either, let's leave as is

**Pull request closed without merging**